src/app/components/countriesCard/fetchCountriesData.tsx

```diff
@@ -4,7 +4,7 @@ import { useState } from 'react';
 import { ChevronDown } from 'lucide-react';
 import { RegionList } from './RegionList';
 import CountryList from './countryList';
-import CountrySearch from './countrySearch';
+import CountrySearch from '../countriesCard/countrySearch';
```

Comment:
CountrySearch MUST NOT be placed in the countriesCard folder. This is because the search functionality belongs to the page itself and not to a country card

Reply:
For this reason, countrySearch should be directly inside the components folder